### DISCUSSION TITLE
is their a version for vue 3?

### DISCUSSION THREAD

**Discussion by JenuelDev** [heroui-inc/heroui#1526] [Q&A]
NextUI really looks good and its beautiful.. is their a support for vue 3?

**Comment by 50l3r** [comment]
> NextUI really looks good and its beautiful.. is their a support for vue 3?

See this: https://nextui.org/docs/guide/introduction#can-i-use-nextui-with-other-front-end-frameworks-or-libraries-such-as-vue-or-angular

U can talk here: https://github.com/nextui-org/nextui/discussions/790

**Comment by wingkwong** [comment]
duplicate - #790
